ruby_review_week_four.rb

```diff
@@ -26,12 +73,84 @@
 
 # b. on a second line, give that fish object’s name attribute the value of ‘Fred’ (assume you have access).
 
+# p fish1 = Fish.new('Fred', 'goldsfish')
+
 # 12. Define a hash called ‘student’. This hash should have two keys, one called name, and the other called age, and assign a string to the name key, and an integer to the age key. 
 
+# student = {
+#   name: "Zenaida", 
+#   age: 8
+# }
+
 # 13. Write a Ruby method that takes three arguments and returns the sum (total) of those three arguments.
 
+# def add(a, b, c)
+#   return a + b + c
+# end 
+
+# p add(2, 3, 4)
+
 # 14. Write a Ruby method that takes an array and a string as arguments, and returns true if the string is in the array, and false if it is not.
 
+# array = [3]
+
+# def array_or_not(array)
+#   if array == ""
+#     return true
+#   else 
+#     return false
+#   end 
+# end 
+
+
+#I think my answer above wasn't answering the question - so I had anoter go below
+array = ["hello"]
+string = "bad"
+def array_or_not(array, string)
+  array.each do |word|
+    if word == string
+      return true
+    else 
+      return false
```

Comment:
this isn't quite right but it's on the right track, we need to think about where the `return false` is located